src/Uno.UI/UI/Xaml/Data/Binding.cs

```diff
@@ -203,6 +224,24 @@ internal void SetBindingXBindProvider(object compiledSource, Func<object, object
 			XBindPropertyPaths = propertyPaths;
 			XBindBack = xBindBack;
 		}
+
+		/// <summary>
+		/// Determines if the FallbackValue has been set
+		/// </summary>
+		/// <remarks>To be used for x:Bind only</remarks>
+		internal bool IsFallbackValueSet
+			=> _flags.HasFlag(BindingFlags.FallbackValueSet);
+
+		[Flags]
```

Comment:
Curious why the enum is attributed with `Flags`? It looks like it's either `None` or `FallbackValueSet`, and `None | FallbackValueSet` is meaningless.

```suggestion
			=> _flags == BindingFlags.FallbackValueSet;

```

Reply:
Ignore this suggestion if the intention is to extend this enum in future with other power-of-two values.

Reply:
The intention is indeed to extend this when more flags will be included.